package/src/utilities/watch-directory.ts

```diff
@@ -1,6 +1,6 @@
 import { readdirSync, statSync } from "node:fs";
 import { join, relative, resolve } from "pathe";
-import { defineUtility } from "../core/define-utility.js";
+import { defineUtility } from "../core/define-utility.ts";
```

Comment:
It should be `.js` though, is that breaking anything?

Reply:
yeah that's a mistake! VSC rewrote it automatically